src/components/ComboBox/ComboBox.tsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import classnames from 'classnames';
+import { getCommonProps } from '../../utils';
+import { Command, CommandInput, CommandList, CommandItem, CommandGroup } from 'cmdk';
+import { Icon } from '../Icon';
+import * as Popover from '@radix-ui/react-popover';
+export interface ComboBoxProps {
+  /**
+   * List of options to be displayed in the ComboBox.
+   */
+  options: {
+    label?: string;
+    value: string;
+  }[];
+  /**
+   * Unique id for the ComboBox.
+   */
+  id: string;
+  /**
+   * Optional className for custom styling.
+   */
+  className?: string;
+  /**
+   * Label for the ComboBox.
+   */
+  label: string;
```

Comment:
can we align with the other input prop names? I believe this is `labelText`. Both our input and select have a `hideLabel` prop that we should probably support here as well.